packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx

```diff
@@ -162,15 +172,27 @@ export function MyJobsTableMobile({
                     }}
                   >
                     {d.url ? (
-                      <TableButton
-                        component={Link}
-                        disabled={buttonDisabled}
-                        fullWidth
-                        target="_blank"
-                        to={d.url}
-                      >
-                        {t('worker.jobs.solve')}
-                      </TableButton>
+                      <>
+                        <TableButton
+                          component={Link}
+                          disabled={buttonDisabled}
+                          fullWidth
+                          target="_blank"
+                          to={d.url}
+                        >
+                          {t('worker.jobs.solve')}
+                        </TableButton>
+                        <RejectButton
```

Comment:
@KacperKoza343 probably something went wrong while resolving conflicts and now we have 2 "reject" buttons

![Uploading Screenshot 2024-10-25 at 13.32.45.png…]()

Reply:
done